Yosemite/Yosemite/Stores/PaymentGatewayAccountStore.swift

```diff
@@ -64,13 +70,34 @@ private extension PaymentGatewayAccountStore {
             case .success(let wcpayAccount):
                 let account = wcpayAccount.toPaymentGatewayAccount(siteID: siteID)
                 self.upsertStoredAccountInBackground(readonlyAccount: account)
-                onCompletion(.success(()))
-                return
-            case .failure(let error):
+            case .failure(_):
+                DDLogDebug("Error fetching WCPay account - it is possible the extension is not installed or inactive")
```

Comment:
nit: is it worth printing the `error` in the debug log? if not, we can remove the underscore:

```suggestion
            case .failure:
                DDLogDebug("Error fetching WCPay account - it is possible the extension is not installed or inactive")
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5734/commits/e7a7ef9406e14f6e9fd5cc10744ce587dbc5fc32 (2 places)